### PR TITLE
 Add Job Name field in AjaxSpider Automation dialogue 

### PR DESCRIPTION
**Pull request by amankumarchagti**

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

File: addOns/spiderAjax/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+  - Add Job Name field in AjaxSpider Automation dialogue
```
Comment (thc202) [inline addOns/spiderAjax/CHANGELOG.md:8]: `AjaxSpider` → `AJAX Spider`

Comment (amankumarchagti) [reply addOns/spiderAjax/CHANGELOG.md]: Done

**Comment by amankumarchagti** [comment]
@kingthorin @thc202 @psiinon  Any other Changes Required??

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**